src/database/sql/bundles/import.sql

```diff
@@ -1,43 +1,58 @@
 -- upsertBundle
 INSERT INTO bundles (
-  id, root_transaction_id, format_id,
-  unbundle_filter_id, index_filter_id,
-  data_item_count, matched_data_item_count,
-  first_queued_at, last_queued_at,
-  first_skipped_at, last_skipped_at,
-  first_unbundled_at, last_unbundled_at,
-  first_fully_indexed_at, last_fully_indexed_at,
-  import_attempt_count, duplicated_data_item_count
+id, root_transaction_id, format_id,
+unbundle_filter_id, index_filter_id,
+previous_unbundle_filter_id, previous_index_filter_id,
+data_item_count, matched_data_item_count,
+first_queued_at, last_queued_at,
+first_skipped_at, last_skipped_at,
+first_unbundled_at, last_unbundled_at,
+first_fully_indexed_at, last_fully_indexed_at,
+import_attempt_count, duplicated_data_item_count
 ) VALUES (
-  @id, @root_transaction_id, @format_id,
-  @unbundle_filter_id, @index_filter_id,
-  @data_item_count, @matched_data_item_count,
-  @queued_at, @queued_at,
-  @skipped_at, @skipped_at,
-  @unbundled_at, @unbundled_at,
-  @fully_indexed_at, @fully_indexed_at,
-  CASE WHEN (@queued_at IS NOT NULL OR @skipped_at IS NOT NULL) THEN 1 ELSE 0 END,
-  @duplicated_data_item_count
+@id, @root_transaction_id, @format_id,
+@unbundle_filter_id, @index_filter_id,
+NULL, NULL,
+@data_item_count, @matched_data_item_count,
+@queued_at, @queued_at,
+@skipped_at, @skipped_at,
+@unbundled_at, @unbundled_at,
+@fully_indexed_at, @fully_indexed_at,
+CASE WHEN (@queued_at IS NOT NULL OR @skipped_at IS NOT NULL) THEN 1 ELSE 0 END,
+@duplicated_data_item_count
 ) ON CONFLICT DO UPDATE SET
-  data_item_count = IFNULL(@data_item_count, data_item_count),
-  matched_data_item_count = IFNULL(@matched_data_item_count, matched_data_item_count),
-  duplicated_data_item_count = IFNULL(@duplicated_data_item_count, duplicated_data_item_count),
-  unbundle_filter_id = IFNULL(@unbundle_filter_id, unbundle_filter_id),
-  index_filter_id = IFNULL(@index_filter_id, index_filter_id),
-  first_queued_at = IFNULL(first_queued_at, @queued_at),
-  last_queued_at = IFNULL(@queued_at, last_queued_at),
-  first_skipped_at = IFNULL(first_skipped_at, @skipped_at),
-  last_skipped_at = IFNULL(@skipped_at, last_skipped_at),
-  first_unbundled_at = IFNULL(first_unbundled_at, @unbundled_at),
-  last_unbundled_at = IFNULL(@unbundled_at, last_unbundled_at),
-  first_fully_indexed_at = IFNULL(first_fully_indexed_at, @fully_indexed_at),
-  last_fully_indexed_at = @fully_indexed_at,
-  import_attempt_count = CASE
+data_item_count = IFNULL(@data_item_count, data_item_count),
+matched_data_item_count = IFNULL(@matched_data_item_count, matched_data_item_count),
+duplicated_data_item_count = IFNULL(@duplicated_data_item_count, duplicated_data_item_count),
+previous_unbundle_filter_id = CASE
+    WHEN @unbundle_filter_id IS NOT NULL THEN @unbundle_filter_id
```

Comment:
Is this logic correct? It appears this will set `previous_unbundle_filter_id` to the value currently being passed into the SQL statement.

Reply:
https://github.com/ar-io/ar-io-node/pull/289/commits/89fabe0f1e734d0bad7129917836224e02bd5454